MapleServer2/Tools/GameCommandActions.cs

```diff
@@ -14,9 +14,21 @@ public static class GameCommandActions
     {
         public static void Process(GameSession session, string command)
         {
-            string[] args = command.ToLower().Split(" ", 2);
+            string[] args = command.ToLower().Split(" ");
             switch (args[0])
             {
+                case "setvalor":
+                    session.Player.Wallet.ValorToken.SetAmount(ParseLong(session, args.Length > 1 ? args[1] : ""));
```

Comment:
this code is repeated many times. it seems like it's doing it for all cases so you can just move it out.

Reply:
Sorry, you mean remove the command?

Reply:
You can't move it out because it's calling it on a different currency each case

Reply:
long amount = ParseLong(session, args.Length > 1 ? args[1] : ""));
...
...SetAmount(amount);

Reply:
Never mind, I'm wrong about how these commands work.